public/javascript/index.js

```diff
@@ -1,23 +1,159 @@
-const charactersAPI = new APIHandler('http://localhost:8000');
+const charactersAPI = new APIHandler('https://minions-api.herokuapp.com/');
+
+
+function characterTemplate(id, name, occupation, cartoon, weapon) {
+	return `<div class="character-info">
+		<div class="id">${id}</div>
+		<div class="name">${name}</div>
+		<div class="occupation">${occupation}</div>
+		<div class="cartoon">${cartoon}</div>
+		<div class="weapon">${weapon}</div>
+	</div>`
+}
```

Comment:
Muy interesante, haciendo uso de técnicas más actuales:

```suggestion
const characterTemplate = (elm) => {
      const {id, name, occupation, cartoon, weapon} = elm
	return `<div class="character-info">
		<div class="id">${id}</div>
		<div class="name">${name}</div>
		<div class="occupation">${occupation}</div>
		<div class="cartoon">${cartoon}</div>
		<div class="weapon">${weapon}</div>
	</div>`
}
```